app/layout/project/sidebar/project/header/project-button/index.tsx

```diff
@@ -19,6 +19,8 @@ import { cn } from 'utils/cn';
 
 import DELETE_WARNING_SVG from 'svgs/notifications/delete-warning.svg?sprite';
 import DOTS_SVG from 'svgs/ui/dots.svg?sprite';
+import COMMUNITY_SVG from 'svgs/ui/new-layout/community.svg?sprite';
+import DOWNLOAD_SVG from 'svgs/ui/upload.svg?sprite';
```

Comment:
Nice try rotating this one 😂 , but we already have a download icon, `svgs/ui/download.svg`, use it please